README.md

```diff
@@ -42,11 +42,24 @@ Change history can be found in [`HISTORY.md`](https://github.com/mongodb/node-mo
 
 ### Compatibility
 
-For version compatibility matrices, please refer to the following links:
+For server and runtime version compatibility matrices, please refer to the following links:
 
 - [MongoDB](https://www.mongodb.com/docs/drivers/node/current/compatibility/#mongodb-compatibility)
 - [NodeJS](https://www.mongodb.com/docs/drivers/node/current/compatibility/#language-compatibility)
 
+#### Component Support Matrix
+
+The following table describes add-on compponent version compatibility for the Node.js driver. Only packages with versions in these supported ranges are stable when used in combination.
+
+| Component                                                                            | `mongodb@3.x` | `mongodb@4.x`      | `mongodb@5.x`      | `mongodb@6.x` |
+| ------------------------------------------------------------------------------------ | ------------- | ------------------ | ------------------ | ------------- |
+| [bson](https://www.npmjs.com/package/bson)                                           | ^1.0.0        | ^2.0.0             | ^5.0.0             | ^6.0.0        |
+| [bson-ext](https://www.npmjs.com/package/bson-ext)                                   | ^1.0.0        | ^2.0.0             | N/A                | N/A           |
+| [kerberos](https://www.npmjs.com/package/kerberos)                                   | ^1.0.0        | ^1.0.0 \|\| ^2.0.0 | ^1.0.0 \|\| ^2.0.0 | ^2.0.0        |
+| [mongodb-client-encryption](https://www.npmjs.com/package/mongodb-client-encryption) | ^1.0.0        | ^1.0.0 \|\| ^2.0.0 | ^2.3.0             | ^6.0.0        |
+| [mongodb-legacy](https://www.npmjs.com/package/mongodb-legacy)                       | N/A           | ^4.0.0             | ^5.0.0             | ^6.0.0        |
+| [@mongodb-js/zstd](https://www.npmjs.com/package/@mongodb-js/zstd)                   | N/A           | ^1.0.0             | ^1.0.0             | ^1.1.0        |
+
 #### Typescript Version
 
 We recommend using the latest version of typescript, however we currently ensure the driver's public types compile against `typescript@4.1.6`.
```

Comment:
"latest version of typescript" => "latest version of TypeScript"
"ensure the driver's" => "ensure the Driver's"

Reply:
Why would "driver" be capitalised? (unless it's a German noun) ;)

Reply:
That's a good point. I guess it depends on if you're referring to the product as the "Node.js Driver", or the "Node.js driver".

When used in isolation (just "driver") it's not the product - just what it is (a driver, or library, or whatever) :)

Feel free to ignore that suggestion

Reply:
For the section heading I'd advocate for changing it to "#### TypeScript Version" as we're referring to TypeScript (the product) and in the text "We recommend using the latest version of `typescript`" (to make it monospaced) since we're referring to the package

Reply:
I'm good with it as is, so I've LGTM'd